src/backend-java-quarkus/src/main/java/ai/azure/openai/rag/workshop/backend/configuration/ChatLanguageModelAzureOpenAiProducer.java

```diff
@@ -4,6 +4,7 @@
 import dev.langchain4j.model.chat.ChatLanguageModel;
 import jakarta.enterprise.inject.Produces;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
+import com.azure.identity.DefaultAzureCredentialBuilder;
```

Comment:
This also need to be added in the "prepare-github-template" script for this file